jasper_reports/JasperReports/browse_data_generator.py

```diff
@@ -320,7 +320,7 @@ def generate(self, file_name):
         reportCopies = self.report.copies or 1
         sequence = 0
         copiesField = self.report.copies_field
-        for record in self.env[self.model].browse(self.ids):
+        for record in self.env['res.users'].browse(self.ids):
```

Comment:
Please do not keep static code.

Reply:
Yes sir, for the testing set the static, I will remove it and make it self.model, after print the report.